eng/scripts/get_package_properties.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import glob
+import os
+
+sys.path.append(os.path.join('scripts', 'devops_tasks'))
+from common_tasks import get_package_properties
+
+def get_all_package_properties():
```

Comment:
Have this function take a path or glob pattern that we can pass to and control the scope.